records_versioning/signals.py

```
@@ -53,10 +55,32 @@ def encounter_saved(sender: object, instance: object, created: bool, **kwargs: o
 def lab_result_saved(sender: object, instance: object, created: bool, **kwargs: object) -> None:
     """Signal receiver for post_save on LabResult model to create a version."""
     _create_version_on_save(instance)
+    # Create a simple AI summary synchronously for tests
+    try:
+        if created:
+            AISummary.objects.create(
+                patient=instance.patient,
+                content_type=ContentType.objects.get_for_model(instance.__class__),
+                object_id=str(instance.pk),
+                summary=f"Lab {getattr(instance, 'loinc', '')}: {getattr(instance, 'value', '')} {getattr(instance, 'unit', '')}"
```

Ruff (0.12.2)

Line too long (129 > 88)

(E501)